src/documentation/attention_manipulation_with_text_controls.ipynb

```diff
@@ -0,0 +1,286 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from aleph_alpha_client import TextControl\n",
+    "from dotenv import load_dotenv\n",
+    "\n",
+    "from intelligence_layer.core import CompleteInput, Llama3InstructModel, NoOpTracer\n",
+    "\n",
+    "load_dotenv()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1",
+   "metadata": {},
+   "source": [
+    "# Attention Manipulation with `TextControl`\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2",
+   "metadata": {},
+   "source": [
+    "`TextControls` enable us to increase or decrease the attention of our model on different parts of the prompt (attention manipulation, AtMan).\n",
```

Comment:
```suggestion
    "`TextControl`s enable us to increase or decrease the attention of our model on different parts of the prompt (attention manipulation, AtMan).\n",
```
maybe? never a big fan of this but not sure how to do this better

Reply:
True